index.js

```diff
@@ -2900,6 +2947,14 @@
 
   return S;
 
+  };
+
+  //  Export two versions of the Sanctuary module: one with type checking;
+  //  one without.
+  var S       = createSanctuary(true);
+  S.unchecked = createSanctuary(false);
+  return S;
+
```

Comment:
I'd love to know your thoughts on this matter of API design, @Avaq, as we discussed this on Gitter.